store/src/java/com/zimbra/qa/unittest/TestScheduledTaskManager.java

```diff
@@ -29,21 +35,23 @@
 import com.zimbra.cs.mailbox.ScheduledTask;
 import com.zimbra.cs.mailbox.ScheduledTaskManager;
 
-public class TestScheduledTaskManager
-extends TestCase {
-
+public class TestScheduledTaskManager {
     static final String TASK_NAME = "TestTask";
     private static final String USER_NAME = "user1";
+    private static Mailbox mbox;
 
-    public void setUp()
-    throws Exception {
-        cleanUp();
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
Please checkout the most recent commit in response to this comment.  If the call is there I have to check for null before calling mbox.getId()